{{ cookiecutter.project_shortname }}/.github/workflows/tests.yml

```diff
@@ -1,95 +1,50 @@
+{% include "misc/header.py" %}
+
 {% raw -%}
 name: CI
 
 on:
   push:
     branches: master
   pull_request:
-    branches: master
+    branches:
+      - master
+      - "maint-**"
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '0 3 * * 6'
+    - cron:  "0 3 * * 6"
   workflow_dispatch:
     inputs:
       reason:
-        description: 'Reason'
+        description: "Reason"
         required: false
-        default: 'Manual trigger'
+        default: "Manual trigger"
 
 jobs:
   Tests:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          # You can add/remove combinations e.g. `dev` requirements or `postgresql13` by adding
-          # a new item to the following lists.
+          # TODO: You can add/remove combinations e.g. `dev` requirements or
+          # `postgresql13` by adding a new item to the following lists.
           # You can see the complete list of services and versions that are available at:
           # https://docker-services-cli.readthedocs.io/en/latest/configuration.html
-          python-version: [3.7, 3.8, 3.9, "3.10"]
-          requirements-level: [min, pypi]
-          cache-service: [redis]
-          db-service: [postgresql10, postgresql13, mysql5, mysql8]
-          mq-service: [rabbitmq, redis]
```

Comment:
- Removed `cache-service` since we only support redis
- Removed `mq-service` since we only support RabbitMQ